src/Popable.tsx

```diff
@@ -223,6 +225,7 @@ const Popable = forwardRef<PopableManager, PopableProps>(function Popable(
             <Popover
               {...sharedPopoverProps}
               forceInitialAnimation
+              caretStyle={caretStyle}
```

Comment:
Could `caretStyle` be added to `sharedPopoverProps` instead?

```js
const sharedPopoverProps = {
  animated,
  ...,
  caretPosition,
  caretStyle, // <<
```

If we don't, `caretStyle` won't be added to the other instance of `Popover` on web below :)